install.ps1

```diff
@@ -222,20 +257,18 @@ if (-not $noChecks.IsPresent) {
         Write-Host "`t[+] Disk is larger than 60 GB" -ForegroundColor Green
     }
 
-    # Check if there is internet connectivity
-    Write-Host "[+] Checking for Internet connectivity..."
-    $connectionStatus =  Test-Connection github.com -Quiet
-    if ($connectionStatus -eq $false)
-    {
-        Write-Host "`t[!] Internet connectivity not detected" -ForegroundColor Red
-        Write-Host "`t[!] Exiting..." -ForegroundColor Red
-        Start-Sleep 3
-        exit 1
-    }
-    else
-    {
-        Write-Host "`t[+] Internet connectivity detected" -ForegroundColor Green
+
+    # Internet connectivity checks
+    Test-WebConnection "google.com"
+    Test-WebConnection "github.com"
+    
+    # Checking for raw.githubusercontent.com connectivity if the URL is in the custom config
+    if ($customConfig -and $customConfig.Contains("raw.githubusercontent.com")) {
```

Comment:
We download the background image from `raw.githubusercontent.com`. So I think we should remove this `if` and test it always.